js/rpg_core/Sprite.js

```diff
@@ -57,13 +57,17 @@ Object.defineProperty(Sprite.prototype, 'bitmap', {
     },
     set: function(value) {
         if (this._bitmap !== value) {
-            this._bitmap = value;
-            if (this._bitmap) {
-                this.setFrame(0, 0, 0, 0);
-                this._bitmap.addLoadListener(this._onBitmapLoad.bind(this));
-            } else {
+            if(!this._bitmap){
+                this._refreshFrame = true;
+            }else if(this._bitmap && value){
```

Comment:
this condition-expression can be written shorter.
Written long for readability?